### PR TITLE
[release/6.0] Backport localizations for UseProgramMain string

### PR DESCRIPTION
**Pull request by captainsafia**
## Description

Copy translations from main to `release/6.0` for top-level statements flag.

Closes https://github.com/dotnet/aspnetcore/issues/41802

## Customer Impact

We're backporting these translations in response to customer feedback that they don't exist on non-English locales.

## Regression?

- [ ] Yes
- [X] No

## Risk

- [ ] High
- [ ] Medium
- [X] Low

## Verification

- [ ] Manual (required)
- [ ] Automated

## Packaging changes reviewed?

- [ ] Yes
- [ ] No
- [X] N/A

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Definitely reviewable 😆

**Comment by captainsafia** [comment]
> Definitely reviewable 😆

Tell me about it! I had to do all these manually. I basically just found the translations in the `main` branch, copied them to the `release/6.0` branch, then updated the key name to "parameters.X." to match the syntax in the `release/6.0` branch.

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/pt-BR/strings.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "version": "1.0.0.0",
   "strings": {
     "name": "Aplicativo Blazor Server",
-    "description": "Um modelo de projeto para criar um aplicativo Blazor Server que é executado do lado do servidor em um aplicativo ASP.NET Core e manipula as interações com o usuário em uma conexão SignalR. Esse modelo pode ser usado para aplicativos Web com UIs (interfaces do usuário) completas e dinâmicas."
+    "description": "Um modelo de projeto para criar um aplicativo Blazor Server que é executado do lado do servidor em um aplicativo ASP.NET Core e manipula as interações com o usuário em uma conexão SignalR. Esse modelo pode ser usado para aplicativos Web com UIs (interfaces do usuário) completas e dinâmicas.",
+    "parameter.UseProgramMain.name": "Não use ins_truções de nível superior",
+    "parameter.UseProgramMain,description": "Se deve gerar uma classe de Programa explícita e um método principal em vez de instruções de nível superior."
```
[the remark this answers is not in the document]
Comment (captainsafia) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/pt-BR/strings.json]: I don't believe so according to some of the other strings that are localized in this branch.

**Review by BrennanConroy — APPROVED** [review]
I managed to successfully review this 😄

**Comment by captainsafia** [comment]
> I managed to successfully review this 😄

Thank you, most graciously! 🙇🏽

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/pt-BR/strings.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dougbu) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/pt-BR/strings.json]: The keys should be identical in all translation files. This is a good fix.

**Comment by dougbu** [comment]
Branch is open but needs servicing approval. Has that happened❔

**Comment by Pilchie** [comment]
@dotnet/aspnet-admins can we try to get this merged today for 6.0.10.

**Comment by dougbu** [comment]
> https://github.com/orgs/dotnet/teams/aspnet-admins can we try to get this merged today for 6.0.10.

We can do it but need the build to be green or at least orange.

**Comment by captainsafia** [comment]
> > https://github.com/orgs/dotnet/teams/aspnet-admins can we try to get this merged today for 6.0.10.
> 
> We can do it but need the build to be green or at least orange.

Thanks for kicking the build. Mind setting it to automerge?

**Comment by dougbu** [comment]
> Mind setting it to automerge?

As you wish

**Pull request merged**